src/ethereum_clis/clis/ethrex.py

```diff
@@ -0,0 +1,81 @@
+"""Ethrex execution client transition tool."""
+
+from ethereum_test_exceptions import BlockException, ExceptionMapper, TransactionException
+
+
+class EthrexExceptionMapper(ExceptionMapper):
+    """Ethrex exception mapper."""
+
+    mapping_substring = {
+        TransactionException.PRIORITY_GREATER_THAN_MAX_FEE_PER_GAS: (
+            "(?i)priority fee is greater than max fee"
+        ),
+        TransactionException.TYPE_3_TX_MAX_BLOB_GAS_ALLOWANCE_EXCEEDED: (
+            "Exceeded MAX_BLOB_GAS_PER_BLOCK"
+        ),
+        TransactionException.TYPE_4_EMPTY_AUTHORIZATION_LIST: "(?i)empty authorization list",
+        TransactionException.INVALID_DEPOSIT_EVENT_LAYOUT: (
+            "Requests hash does not match the one in the header after executing"
+        ),
```

Comment:
This feels to me like there's a bug in the client, it should not get to the point of making the comparison of request hashes.

This PR might be missing from the implementation IMO: https://github.com/ethereum/EIPs/commit/619221e382064e3caad8fac96d626079e9308d23

Reply:
Yes, that was indeed the case! Thanks for pointing it out!
We have a fix ready so I pushed the updated error mapping